src/App.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import Plus from './templates/add.svg';
+import {isEmpty} from 'lodash';
```

Comment:
there are convention through this project, like `{ something }`, not `{something}`.
`{ something }` is much more readable. Please improve that through whole project (I remember the same issue with importing action types).